src/styles.css

```
@@ -707,3 +707,99 @@ footer a:hover{text-decoration:underline}
+    0 10px 25px rgba(244, 107, 14, 0.15);
+  border-color: var(--orange-border);
+}
+
+/* Floating light glow layer */
+.org-card::before {
+  content: "";
+  position: absolute;
+  inset: 0;
+  background: radial-gradient(
+    circle at var(--x, 50%) var(--y, 50%),
+    rgba(244, 107, 14, 0.25),
```

Stylelint (17.11.0)

[error] Expected "rgba" to be "rgb" (color-function-alias-notation)

(color-function-alias-notation)